WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -120,19 +115,23 @@ public void loadPost() {
         protected Spanned doInBackground(Void... params) {
             Spanned contentSpannable;
 
-            if (mActivity == null || mActivity.getPost() == null) {
+            if (getActivity() == null) {
                 return null;
             }
 
-            Post post = mActivity.getPost();
+            PostModel post = ((EditPostActivity) getActivity()).getPost();
```

Comment:
Done in 123dc2d. Also did the same thing for `EditPostSettingsFragment` in 24589f5, and eliminated `EditPostActivity.getPost()` entirely in 21a34bd to remove future temptation.